src/lib/_imports/components/cortal.icon.css

```diff
@@ -43,9 +43,10 @@ Styleguide Components.Cortal.Icon
 /* To tweak action icon appearance and layout */
 :--c-card > :is(:--action) > .icon:first-child,
 p > :is(:--action):only-child > .icon:first-child {
-    vertical-align: middle;
     margin-right: 10px;
     font-size: 1.2em;
```

Comment:
View notes in description for this. Consider commenting out, or losing it altogether. Lmk and I'll make the change.

Reply:
Line 47.

Reply:
Thanks for calling this out. Please keep the font-size. It is correct, and should have effect. That it does not have effect is a bug. _The bug is caused (at least on prod) by `imports-….css` having global effect atop CMS code. I do not know a fix yet._

<details>

These screenshots are without your fix, and are a comment on the font-size, not a comment on your code changes.

| what icon should look like | what icon does look like |
| - | - |
| ![showing icon at correct size - future](https://github.com/TACC/Core-Styles/assets/62723358/09f29a02-d6a5-486b-87a4-de70d81873a5) | ![showing icon at wrong size - current](https://github.com/TACC/Core-Styles/assets/62723358/d1e4494d-55fa-45f3-986d-dd0abac8dad0) |

I thought the bug was always, but your "Testing" notes[^1] suggests local and prod behave differently.

> **Note**: When testing, localhost:8000 will use the 1.2em attribute. However, TACC production site uses font-size: 1em/1 cortal."

</details>

Reply:
**Hm, actually…** Please check with designers. _I see the icon is rather small in [the original design](https://xd.adobe.com/view/dabaec5a-cc31-4c5c-876e-56d6d8cc359e-8fe9/screen/032918ef-38c1-4cd2-bd3b-c38a5af710da/specs/). Maybe designers wanted it bigger later, or maybe the font-size is my mistake._

Reply:
[We got a private approval for font-size `1.2` from designers.](https://tacc-team.slack.com/archives/C060NU133D5/p1699986327348609)

It's not working in prod, but that's related to [the bug I screenshotted](https://github.com/TACC/Core-Styles/pull/260/files#r1391805647).

I'm marking this resolved, but feel free to re-open if there's more to it.